src/main/java/no/difi/move/kosmos/service/mail/MailServiceImpl.java

```diff
@@ -29,14 +29,19 @@ public void sendMail(String subject, String content) {
         }
         Objects.requireNonNull(mail.getRecipient(), "kosmos.mail.recipient must be set");
         Objects.requireNonNull(mail.getFrom(), "kosmos.mail.from must be set");
+        String appendSubject = properties.getMail().getAppendSubject();
+        if (appendSubject != null && !appendSubject.isEmpty()) {
+            subject += " - " + appendSubject;
+        }
         log.trace("Sending email from {} to {} with subject '{}'", mail.getFrom(), mail.getRecipient(), subject);
+        String finalSubject = subject;
```

Comment:
Denne trengs vel ikkje?

Reply:
"variable used in lambda expression should be final or effectively final" seie den om eg setSubject(subject); direkte.